package.json

```diff
@@ -22,6 +22,8 @@
   "scripts": {
     "test-local": "wdio ./config-wdio/wdio.local.conf.ts",
     "test-dev": "SET NODE_ENV=dev&& npx wdio ./config-wdio/wdio.local.conf.ts",
+    "test-dev-smoke": "SET NODE_ENV=dev&& npx wdio ./config-wdio/wdio.local.conf.ts --mochaOpts.grep=@Smoke",
```

Comment:
Can we have a suite name 'regression or adhoc' which responsible for both sanity and smoke test case running

Reply:
Chandra, this is for reference for others to follow, we can meet and segregate the test cases later.